indico/modules/rb_new/client/js/components/TimelineBase.jsx

```diff
@@ -175,9 +99,9 @@ export default class TimelineBase extends React.Component {
 
     render() {
         return (
-            <Container>
+            <div>
```

Comment:
Have you checked whether this `<div/>` is even needed? If not, we could probably do OK with renaming `renderContent()` => `render()`.